From 498fbdbe081260f7e42a1659e11912f9f2c0e5ad Mon Sep 17 00:00:00 2001
Message-Id: <498fbdbe081260f7e42a1659e11912f9f2c0e5ad@dist-git>
From: Peter Krempa <pkrempa@redhat.com>
Date: Tue, 4 Apr 2017 09:31:21 +0200
Subject: [PATCH] qemu: hotplug: Fix formatting strings in
 qemuDomainFilterHotplugVcpuEntities

'next' is declared as 'ssize_t' so use '%zd'

(cherry picked from commit 86d69c309181ed7782427261abd85a4b3fdc704f)

https://bugzilla.redhat.com/show_bug.cgi?id=1437010
---
 src/qemu/qemu_hotplug.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/qemu/qemu_hotplug.c b/src/qemu/qemu_hotplug.c
index b5b520d8c..48de6b815 100644
--- a/src/qemu/qemu_hotplug.c
+++ b/src/qemu/qemu_hotplug.c
@@ -5823,13 +5823,13 @@ qemuDomainFilterHotplugVcpuEntities(virDomainDefPtr def,
 
         if (vcpu->online == state) {
             virReportError(VIR_ERR_INVALID_ARG,
-                           _("vcpu '%zu' is already in requested state"), next);
+                           _("vcpu '%zd' is already in requested state"), next);
             goto cleanup;
         }
 
         if (vcpu->online && !vcpu->hotpluggable) {
             virReportError(VIR_ERR_INVALID_ARG,
-                           _("vcpu '%zu' can't be hotunplugged"), next);
+                           _("vcpu '%zd' can't be hotunplugged"), next);
             goto cleanup;
         }
     }
@@ -5845,7 +5845,7 @@ qemuDomainFilterHotplugVcpuEntities(virDomainDefPtr def,
 
         if (vcpupriv->vcpus == 0) {
             virReportError(VIR_ERR_INVALID_ARG,
-                           _("vcpu '%zu' belongs to a larger hotpluggable entity, "
+                           _("vcpu '%zd' belongs to a larger hotpluggable entity, "
                              "but siblings were not selected"), next);
             goto cleanup;
         }
@@ -5854,7 +5854,7 @@ qemuDomainFilterHotplugVcpuEntities(virDomainDefPtr def,
             if (!virBitmapIsBitSet(map, i)) {
                 virReportError(VIR_ERR_INVALID_ARG,
                                _("vcpu '%zu' was not selected but it belongs to "
-                                 "hotpluggable entity '%zu-%zu' which was "
+                                 "hotpluggable entity '%zd-%zd' which was "
                                  "partially selected"),
                                i, next, next + vcpupriv->vcpus - 1);
                 goto cleanup;
-- 
2.12.2

